package.json

```diff
@@ -154,9 +154,13 @@
 		"pnpm": "7"
 	},
 	"pnpm": {
+		"comments": [
+			"nodegit is replaced with an empty package here because it's currently only used by good-fences for features we do not need, and has issues building when changing node versions."
```

Comment:
Might be nice to link either of the GH issues in your description in this comment--gives future changes a bit more context on what the actual blockers are here and potential other options (for example if we want to reconsider if your linked issue is resolved)

Reply:
DOne.